https://github.com/libofx/libofx/pull/77

From 55923eba7617c7c4056b21ddf3b569adaea43858 Mon Sep 17 00:00:00 2001
From: Sam James <sam@gentoo.org>
Date: Tue, 6 Sep 2022 06:00:04 +0100
Subject: [PATCH 2/2] autotools: fix parallel build issue with ofxconnect,
 ofxdump

We need to build the relevant tool before calling help2man, as
help2man tries to call the tool itself.

Otherwise, we get:
```
help2man: can't get `--help' info from ./ofxdump
Try `--no-discard-stderr' if option outputs to stderr
make[2]: *** [Makefile:1211: ofxdump.1] Error 127
make[2]: *** Waiting for unfinished jobs....
```

Ends up building okay as a workaround with -j1.

Bug: https://github.com/libofx/libofx/pull/56
See: 0597b8a0a9da3226af0779163fbb3ca389d70bc9
Fixes: 76dae192b4bf642b311084043cf9d6273dd69bb2
Signed-off-by: Sam James <sam@gentoo.org>
--- a/ofxconnect/Makefile.am
+++ b/ofxconnect/Makefile.am
@@ -23,9 +23,9 @@ EXTRA_DIST = cmdline.ggo test-privateserver.sh CMakeLists.txt
 
 # See README.privateserver for details on this server and how to get
 # the key needed to run this test.
-TESTS = test-privateserver.sh 
+TESTS = test-privateserver.sh
 
-ofxconnect.1: $(top_srcdir)/configure.ac
+ofxconnect.1: ofxconnect$(EXEEXT)
 if HAVE_HELP2MAN
 	$(HELP2MAN)  -n 'Create a statement request file' -N --output=ofxconnect.1 ./ofxconnect$(EXEEXT)
 else
--- a/ofxdump/Makefile.am
+++ b/ofxdump/Makefile.am
@@ -15,7 +15,7 @@ cmdline.c cmdline.h: cmdline.ggo Makefile
 
 endif
 
-ofxdump.1: $(top_srcdir)/configure.ac
+ofxdump.1: ofxdump$(EXEEXT)
 if HAVE_HELP2MAN
 	$(HELP2MAN) -n 'Dump content of OFX files as human-readable text' -N --output=ofxdump.1 ./ofxdump$(EXEEXT)
 else
